src/runtime/Makefile

```diff
@@ -121,6 +122,9 @@ DEFROOTFSTYPE := $(ROOTFSTYPE_EXT4)
 FIRMWAREPATH :=
 FIRMWAREVOLUMEPATH :=
 
+FIRMWARETDVFPATH := $(PREFIXDEPS)/share/tdvf/OVMF.fd
```

Comment:
hmm.... the merge from CCv0 to main has a potential to turn this Makefile more convoluted than it is today. We should start thinking on how to break it into smaller makefile snippets to be included in.

Reply:
I have been going back and forth on this, mainly on a way to have a different snippet that could only be loaded specifically for a runtime-class, but I sincerely don't see how it can happen unless we:
* symlink the default configuration
* create snippets under that new name

The reason for that is because once a snippet is in, it'll be loaded, and right now I do believe users will still want to have the "vanilla" snippets and the TEEs ones. :-/